cernopendata/templates/cernopendata_pages/front/lva.html

```diff
@@ -0,0 +1,91 @@
+<div class="homepage-section section lva align-items-center" id="lva" data-anchor="lva">
+    <div class="main-inner container-fluid align-items-center">
+        <div class="row align-items-center justify-content-around">
+                <div class="col-sm lva-item">
+                    <header class="container-fluid">
+                        <h3 class="row justify-content-center">Learn</h3>
+                        <div class="row justify-content-center">
```

Comment:
Can we force the boxes to be the same size?  We can also do this anytime later when the content will be updated...
![page-two](https://user-images.githubusercontent.com/517546/33532008-9cde9768-d895-11e7-8fbd-c07b643e6241.png)